app/blueprints/round/templates/round_details.html

```diff
@@ -0,0 +1,539 @@
+{% from "govuk_frontend_jinja/macros/attributes.html" import govukAttributes -%}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{% from "govuk_frontend_jinja/components/table/macro.html" import govukTable %}
+{%- from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary -%}
+{%- from "govuk_frontend_jinja/components/details/macro.html" import govukDetails -%}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{%- from "govuk_frontend_jinja/components/summary-list/macro.html" import govukSummaryList -%}
+{% from "macros/wtfToGovUk.html" import backlink %}
+{% set showNavigationBar = True %}
+{% set active_item_identifier = "applications" %}
+{% extends "base.html" %}
+{% block beforeContent %}
+  {{ super() }}
+     <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            {{ govukBackLink({
+              "text": "Back",
+              "href": url_for("round_bp.view_all_rounds")
+            }) }}
+        </div>
+    </div>
+{% endblock beforeContent %}
+{% block content %}
+<div class="govuk-grid-row">
+   <div class="govuk-grid-column-full">
+    <div class="govuk-grid-column-three-quarters govuk-!-padding-0">
+        <h2 class="govuk-heading-l govuk-!-margin-bottom-0">Apply for {{ round.fund.title_json["en"] }}</h2>
+        <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for("fund_bp.edit_fund", fund_id=round.fund.fund_id, _anchor=fund_form.title_en.id) }}">Change application heading</a>
+    </div>
+    <div class="govuk-grid-column-one-quarter govuk-!-text-align-right">
+        {{ govukButton({ "text": "Build application",
+            "href": url_for("application_bp.build_application", round_id=round.round_id),
+            "classes": "govuk-button--secondary"
+        }) }}
+    </div>
+   </div>
+</div>
+<div class="govuk-grid-row">
+   <div class="govuk-grid-column-full">
```

Comment:
If this just contains normal content it should be a column-two-thirds, but not a big deal right now as none of the content here is long.